tastytaps/taps/urls.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import routers
+
+from . import api
+
+
+router = routers.SimpleRouter()
+
+router.register(r'brewery', api.BreweryViewSet)
+router.register(r'price', api.PriceViewSet)
+router.register(r'taps', api.TapsViewSet)
```

Comment:
I agree. We should be we consistent. I'll make the urls plural (taps, prices and breweries)